samples-final/AspNetCore-LUIS-Bot/LUIS-Reminders.json

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
I think this is great, but too complex. 

Can we cut this waaaaaay down?

Reply:
This sample uses the same LUIS model in nodeJS, however, we will simplify this model to only keep the `Calendar.Add` (without entity recognition) and `Calendar.Find` intents.